.github/ISSUE_TEMPLATE/webtestplan.md

```diff
@@ -479,412 +624,454 @@ Add the following to enable read access to the auth connectors
+  - Upload
+    - [ ] Verify if uploading single/multiple files works.
+    - [ ] Verify that invalid paths and network errors are handled.
+    - [ ] Verify if cancelling the upload works.
+- Refreshing certs
+  - To test scenarios from this section, create a user with a role that has TTL of `1m`
+    (`spec.options.max_session_ttl`).
+  - Log in, create a db connection and run the CLI command; wait for the cert to expire, make
+    another connection to the local db proxy.
+    - [ ] Verify that the window received focus and a modal login is shown.
       - Verify that this works on:
-         - [ ] macOS
-         - [ ] Windows
-         - [ ] Linux
-   - Change a config property and restart the app. Verify that the change has been applied.
-      - [ ] Change a keyboard shortcut.
-      - [ ] Change `terminal.fontFamily`.
-   - Provide the same keyboard shortcut for two actions.
-      - [ ] Verify that a notification is displayed saying that a duplicate shortcut was found.
-   - Provide an invalid value for some property (for example, set `"keymap.tab1": "ABC"`).
-      - [ ] Verify that a notification is displayed saying that the property has an invalid value.
-   - Make a syntax error in the file (for example, set `"keymap.tab1": not a string`).
-      - [ ] Verify that a notification is displayed saying that the config file was not loaded correctly.
-      - [ ] Verify that your config changes were not overridden.
+        - [ ] macOS
+        - [ ] Windows
+        - [ ] Linux
+    - Verify that after successfully logging in:
+      - [ ] The cluster info is synced.
+      - [ ] The first connection wasn't dropped; try executing `select now();`, the client should
+            be able to automatically reinstantiate the connection.
+      - [ ] The database proxy is able to handle new connections; click "Run" in the db tab and
+            see if it connects without problems. You might need to resync the cluster again in case
+            they managed to expire.
+    - [ ] Verify that closing the login modal without logging in shows an appropriate error.
+  - Log in, create a db connection, then remove access to that db server for that user; wait for
+    the cert to expire, then attempt to make a connection through the proxy; log in.
+    - [ ] Verify that psql shows an appropriate access denied error ("access to db denied. User
+          does not have permissions. Confirm database user and name").
+  - Log in, open a cluster tab, wait for the cert to expire. Switch from a servers view to
+    databases view.
+    - [ ] Verify that a login modal was shown.
+    - [ ] Verify that after logging in, the database list is shown.
+  - Log in, set up two db connections. Wait for the cert to expire. Attempt to connect to the first
+    proxy, then without logging in proceed to connect to the second proxy.
+    - [ ] Verify that an error notification was shown related to another login attempt being in
+          progress.
```

Comment:
This seems wrong, each of those main bullet points is duplicated in the Connect test plan. I haven't checked how the points added here differ from those that were already there. But this should be pretty clear once the new points are moved to the correct location.